static/js/flow/reduxTypes.js

```diff
@@ -13,3 +13,8 @@ export type AsyncActionHelper = (...a: any) => Promise<*>;
 export type ActionCreator = (...a: any) => Action;
 
 export type AsyncActionCreator<T> = (...a: any) => Dispatcher<T>;
+
+export type Store = {
+  dispatch: Dispatch,
+  getState: () => any,
+};
```

Comment:
We have an annotation lib file for redux (in `flow-typed`) which might have something that covers this already.